swift/codegen/templates/trap_traps.mustache

```diff
@@ -29,5 +29,14 @@ inline std::ostream &operator<<(std::ostream &out, const {{name}}{{trap_affix}}
       << {{#get_streamer}}e.{{field_name}}{{/get_streamer}}{{/fields}} << ")";
   return out;
 }
+{{#id}}
+
+namespace detail {
+template<>
+struct ToBindingTrapFunctor<{{type}}> {
+  using type = {{name}}{{trap_affix}};
+};
+}
+{{/id}}
```

Comment:
Am I correct that the generated QL code is included in the repo (for obvious reasons), but the generated C++ code is not (it's just generated during the build when needed)?

Reply:
yes, you are right. However in a PR you can consult the generated headers by going to the summary of the `Swift: Check code generation / codegen` action run ([here](https://github.com/github/codeql/actions/runs/2356916003) for this PR at the moment)

Reply:
(FYI it does require 2 clicks from the PR discussion, first on the run `details`, and then on the `Summary` at the left, which might not be super intuitive)